Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.mm

```diff
@@ -107,19 +107,19 @@
 
 void RemoteLayerTreeDrawingAreaProxy::viewWillStartLiveResize()
 {
-    if (auto scrollingCoordinator = m_webPageProxy.scrollingCoordinatorProxy())
+    if (auto scrollingCoordinator = m_webPageProxy->scrollingCoordinatorProxy())
         scrollingCoordinator->viewWillStartLiveResize();
 }
 
 void RemoteLayerTreeDrawingAreaProxy::viewWillEndLiveResize()
 {
-    if (auto scrollingCoordinator = m_webPageProxy.scrollingCoordinatorProxy())
+    if (auto scrollingCoordinator = m_webPageProxy->scrollingCoordinatorProxy())
         scrollingCoordinator->viewWillEndLiveResize();
 }
 
 void RemoteLayerTreeDrawingAreaProxy::deviceScaleFactorDidChange()
 {
-    m_webPageProxy.send(Messages::DrawingArea::SetDeviceScaleFactor(m_webPageProxy.deviceScaleFactor()), m_identifier);
+    protectedWebPageProxy()->send(Messages::DrawingArea::SetDeviceScaleFactor(m_webPageProxy->deviceScaleFactor()), m_identifier);
```

Comment:
Here you're using `protectedWebPageProxy()` to send IPC but you're not doing this consistently earlier if your patch.

Reply:
whereas here, send is not a trivial function so we need to store m_webPageProxy in RefPtr.